public/styles/queue.css

```diff
@@ -0,0 +1,114 @@
+.song-queue {
+  position: absolute;
+  min-height: 278px;
+  padding: 1rem;
+  right: 29px;
+  justify-items: center;
+  max-height: 100%;
+  background: rgba(0, 0, 0, 0.1);
+  border: 2px solid rgba(255, 255, 255, 0.1);
+  box-sizing: border-box;
+  backdrop-filter: blur(80px);
+  border-radius: 24px;
+  width: 1px;
+  opacity: 0;
+  transition: all .75s ease;
+  max-height: 400px;
+}
+
+.song-queue.show {
+  opacity: 1;
+  width: 275px;
+  max-height: 400px;
+  min-height: 278px;
+}
+
+.queue-list {
+  display: flex;
+  padding: 5px;
+  position: relative;
+  flex-direction: column;
+  max-height: 300px;
+  overflow-y: scroll;
+}
+
+.queue-group {
+  /* border-bottom: 1px solid grey; */
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: space-between;
+  width: 80%;
+}
+
+.queue-group::after {
+  content: '';
+  position: absolute;
+  width: 100%;
+  height: 1px;
+  bottom: -10px;
+  transform: translate(0, +50%);
+  background-color: white;
+}
+
+.queue-song {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  margin-bottom: 21px;
+}
+
+.queue-song:last-of-type .queue-group::after {
+  display: none;
+}
+
+.queue-album {
+  height: 35px;
+  width: 35px;
+}
+
+.queue-album img {
+  border-radius: 50%;
+  animation: rotation 5s infinite linear;
+}
+
+@keyframes rotation {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(359deg);
+  }
+}
+
+.queue-album-circle {
+  position: relative;
+  z-index: 99;
+  left: 13px; 
+  top: -28px;
+  border-radius: 100%;
+  width: 8px; 
+  height: 8px;
+  background-color: rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(80px);
+}
+
+.queue-title {
+  display: grid;
+  font-weight: 500;
+  color: #F2F2F2;
```

Comment:
Do we have color vars for this?

Reply:
Nope. Just updated the colors!

![Screen Shot 2022-01-17 at 3 23 42 PM](https://user-images.githubusercontent.com/4999022/149847057-6a08b0b2-fc09-4861-baa0-a568e8a05510.png)